src/rules/strictComparisonsRule.ts

```diff
@@ -276,6 +354,26 @@ function isEqualityOperator(node: ts.BinaryExpression): boolean {
     }
 }
 
+function getKindFromSyntaxKind(type: ts.SyntaxKind): TypeKind {
+    // tslint:disable:prefer-switch
```

Comment:
Is there a reason you wanted an `if`/`else` here? The `prefer-switch` rule was enabled on the codebase as a stylistic preference to keep the code more consistent. A `switch` statement, `Map`, or `object` would generally be what we'd prefer.